core/src/main/java/com/linecorp/armeria/server/logging/AccessLogComponent.java

```diff
@@ -277,7 +277,8 @@ public Object getMessage0(RequestLog log) {
                     } else {
                         // %a means the client address who initiated a request.
                         final ServiceRequestContext ctx = (ServiceRequestContext) log.context();
-                        return ctx.clientAddress().getHostAddress();
+                        return ctx.clientAddressMapper().apply(ctx.proxiedAddresses())
+                                  .getHostString();
```

Comment:
Question: I just log `host` only and want to hear other folks' opinions.
* Should we need to log `host:port`? 🤔
* If we need, fix `REMOTE_HOST` and `LOCAL_IP_ADDRESS` too?

Reply:
Address only because most components here stem from a format string specification of some old HTTPd.

Reply:
I revert this. Now `ctx.clientAddress()` is result of `ctx.clientAddressMapper().apply(ctx.proxiedAddresses())`.